pkg/ottl/expression_test.go

```diff
@@ -1449,6 +1449,215 @@ func Test_StandardIntLikeGetter_WrappedError(t *testing.T) {
 	assert.False(t, ok)
 }
 
+func Test_StandardByteSliceLikeGetter(t *testing.T) {
+	tests := []struct {
+		name             string
+		getter           ByteSliceLikeGetter[any]
+		want             any
+		valid            bool
+		expectedErrorMsg string
+	}{
+		{
+			name: "string type",
+			getter: StandardByteSliceLikeGetter[any]{
+				Getter: func(_ context.Context, _ any) (any, error) {
+					return "1", nil
+				},
+			},
+			want:  byteSlice("1"),
+			valid: true,
+		},
+		{
+			name: "byte type",
+			getter: StandardByteSliceLikeGetter[any]{
+				Getter: func(_ context.Context, _ any) (any, error) {
+					return []byte("1"), nil
+				},
+			},
+			want:  byteSlice("1"),
+			valid: true,
+		},
+		{
+			name: "byteSlice type",
+			getter: StandardByteSliceLikeGetter[any]{
+				Getter: func(_ context.Context, _ any) (any, error) {
+					return byteSlice("1"), nil
+				},
+			},
+			want:  byteSlice("1"),
+			valid: true,
+		},
+		{
+			name: "int64 type",
+			getter: StandardByteSliceLikeGetter[any]{
+				Getter: func(_ context.Context, _ any) (any, error) {
+					return int64(12), nil
+				},
+			},
+			want:  byteSlice(int64ToBytes(12)),
```

Comment:
Can we update the `want` values for these tests to have the concrete values we're looking for (i.e. `[]byte{...}`)? I want to make sure we have a concrete idea of what the output values are.